server/openapi_server/models/text_annotation.py

```diff
@@ -152,6 +152,8 @@ def confidence(self, confidence):
         :param confidence: The confidence of this TextAnnotation.
         :type confidence: float
         """
+        if confidence is None:
```

Comment:
Will the confidence be eventually used in scoring?

Reply:
Yes, to compute AUROC and AUPR. The confidence is also to enable tools that rely on the annotator to make decision based on the confidence level.